Content.Shared/_RMC14/Xenonids/Plasma/XenoRemoteTransferPlasmaComponent.cs

```diff
@@ -0,0 +1,19 @@
+using Content.Shared.Actions;
+using Content.Shared.FixedPoint;
+using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared._RMC14.Xenonids.Plasma;
+
+[RegisterComponent, AutoGenerateComponentState, NetworkedComponent]
+public sealed partial class XenoRemoteTransferPlasmaComponent : Component
+{
+    [DataField("Percentage"), AutoNetworkedField]
+    public FixedPoint2 PlasmaPercentage = 0.75;
```

Comment:
Don't specify a data field name, it will use the field name camel-cased automatically (plasmaPercentage)
If you are using this in YAML, make sure to update that too, or just rename the field to Percentage